data.json

```diff
@@ -1384,5 +1384,11 @@
     "githubUsername": "DarshGupta1910",
     "imageUrl": "https://user-images.githubusercontent.com/73016212/135509959-87f8b8e4-9fab-4247-bae4-d965fca5ddbc.png",
     "issueNumber": 742
+  },
+  {
+    "name": "Ekta Arora",
+    "githubUsername": "ektaarora16",
+    "imageUrl": "https://user-images.githubusercontent.com/90129737/135709011-e377fce0-d0b9-46d5-8eff-f2edec70e8fa.png",
```

Comment:
I would recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/135740246-b0be6db0-160a-43d8-819d-0ad3c190619c.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/135740246-b0be6db0-160a-43d8-819d-0ad3c190619c.png",
```